src/VideoManager/SubtitleWriter.cc

```diff
@@ -22,6 +22,15 @@
 #include <QDate>
 
 QGC_LOGGING_CATEGORY(SubtitleWriterLog, "SubtitleWriterLog")
+const QString vehicleFactGroupName = QStringLiteral("Vehicle");
+
+QStringList settingsSources{
+    QStringLiteral("ValuePageUserSettings2"),
+    QStringLiteral("ValuePageDefaultSettings2")
+};
+
+const QString factNameTemplate = QStringLiteral("rows/%1/columns/%2/factName");
+const QString groupNameTemplate = QStringLiteral("rows/%1/columns/%2/groupName");
```

Comment:
can't this variables be in the class and not global ? or why not inside the function scope ?

Reply:
yes, I just haven't tested and pushed the changes yet.